starlingcaptureapi/crypto_util.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import os
+from Crypto.Cipher import AES
+
+_logger = logging.getLogger(__name__)
+
+
+def new_aes_key():
```

Comment:
Is this meant for use on the command line? Would be good to document the intended usage in the docstring, since it seems like it is not going to be used in the code directly.

Reply:
In general, pls try to document public functions.

Reply:
It's possible that when we see a new stream without an encryption key, we'll generate one and store _somewhere_. This may not necessarily be called from CLI. Can keep as is for now?

Reply:
Ok, I will keep this function for now.